src/decision/include/DeadMansSwitchNode.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Created By: Gareth Ellis
+ * Created On: July 22nd, 2018
+ * Description: A node that publishes how long it's been since it received a
+ *              message on a given topic. This can be used by other nodes
+ *              to check if they have died and how long ago. Currently it's
+ *              used by the firmware running on the Arduino to check if the
+ *              safety has previously been triggered (and so cut power to
+ *              the Arduino).
+ */
+
+#ifndef DECISION_DEAD_MANS_SWITCH_NODE_H
+#define DECISION_DEAD_MANS_SWITCH_NODE_H
+
+// ROS Includes
+#include <ros/ros.h>
+#include <std_msgs/Duration.h>
+#include <std_msgs/Empty.h>
+
+class DeadMansSwitchNode {
+  public:
+    DeadMansSwitchNode(int argc, char** argv, std::string node_name);
+
+  private:
+    /**
+     * Callback function for when a new string is received
```

Comment:
picky but it looks like it's not actually getting a string?

Reply:
Oh, good catch, that was carry over. Revamped the docstring.